lib/app_server.py

```diff
@@ -103,7 +105,12 @@ def deploy(self, vardir=None, silent=True, need_init=True):
                     if (e.errno == errno.ENOENT):
                         continue
                     raise
-        os.putenv("LISTEN", str(find_port()))
+        if self.use_unix_sockets_iproto:
+            path = os.path.join(self.vardir, self.name + ".socket-iproto")
+            warn_unix_socket(path)
+            os.putenv("LISTEN", str(path))
```

Comment:
Nit: It is already string, `str()` is not needed.

Reply:
All corrected as you suggested.